### PR TITLE
Add Outdated badge to `hydrogen` & `glacier-darkula-ui`

### PR DESCRIPTION
**Pull request by confused-Techie**
### Requirements 

* Filling out the template is required.
* All new code requires tests to ensure against regressions.
  - However, if your PR contains zero code changes, feel free to select the checkmark below to indicate so.

* [ ] Have you ran tests against this code?
* [X] This PR contains zero code changes.

### Description of the Change

This PR adds the outdated badges to the above packages, while also adding the docs for what the outdated badge means, and how it can be removed.

The listing in Admin Actions here, would be the location the link of the badge points to, so contains full instructions for the end users on how to install the proper version of the package. As well as the full reasoning of why this was added.

### CODE REVIEW

File: docs/reference/badge-spec.md [hunk 1]
```diff
@@ -56,3 +56,17 @@ The following are some valid examples of badges:
...
+
+# Types of Badges & Why They May be Added
+
+## Outdated
+
+The `Outdated` badge is used to indicate that the version of a package that may exist on the Pulsar Package Registry, may be out of date when compared to the package on GitHub (Or other VCS host).
+
+This badge should only be used when the main functionality of the branch is missing/broken/or otherwise non-functional when installed onto Pulsar as the latest branch, in a supported Pulsar configuration.
+
+That is, if a Pulsar users on a supported platform, indepent of any other issues, installs a package and it immediately does not work, displays severe visual bugs, or causes an error message logged as a notification, and there is a fix available for that package within it's source code, that has not been pushed to Pulsar in a reasonable time, then it is elligble to receive this badge.
```
Comment (Meadowsys) [inline docs/reference/badge-spec.md:68]:
```suggestion
That is, if a Pulsar users on a supported platform, indepent of any other issues, installs a package and it immediately does not work, displays severe visual bugs, or causes an error message logged as a notification, and there is a fix available for that package within it's source code, that has not been pushed to Pulsar in a reasonable time, then it is eligible to receive this badge.
```

**Review by Meadowsys — APPROVED** [review]
smol typo, but otherwise looks okay

File: docs/reference/badge-spec.md (hunk 1 above)
Comment (confused-Techie) [reply docs/reference/badge-spec.md:68]: Thanks for finding this!

**Comment by Daeraxa** [comment]
Looks good, we might also want to demote it from the featured packages with this change? Seems weird to publicise a package we deem outdated.

**Review by Daeraxa — APPROVED** [review]

**Comment by confused-Techie** [comment]
Yeah I'll remove it first from featured packages, thanks for thinking of that, would be a little awkward @Daeraxa

**Pull request merged**